providers/alternatives.rb

```diff
@@ -38,8 +38,9 @@
         description = "Add alternative for #{cmd}"
         converge_by(description) do
           Chef::Log.debug "Adding alternative for #{cmd}"
-          shell_out("rm /var/lib/alternatives/#{cmd}")
```

Comment:
When running alternatives commands, you mention the OS will clean up, what do you mean by that? Wont you have orphaned alternative entries if you dont rm it?